exchanges/anx/anx_mock_test.go

```diff
@@ -0,0 +1,46 @@
+//+build !mock_test_off
+
+// This will build if build tag mock_test_off is not parsed and will try to mock
+// all tests in _test.go
+package anx
+
+import (
+	"os"
+	"testing"
+
+	"github.com/thrasher-corp/gocryptotrader/config"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/mock"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/sharedtestvalues"
+	log "github.com/thrasher-corp/gocryptotrader/logger"
+)
+
+const mockFile = "../../testdata/http_mock/anx/anx.json"
+
+var mockTests = true
+
+func TestMain(m *testing.M) {
+	cfg := config.GetConfig()
+	cfg.LoadConfig("../../testdata/configtest.json")
+	anxConfig, err := cfg.GetExchangeConfig("ANX")
+	if err != nil {
+		log.Error("Test Failed - Mock server error", err)
```

Comment:
This can be simplified to log.Fatal, as this automatically calls os.Exit(1) after printing